src/controllers/user.controller.ts

```diff
@@ -85,7 +89,7 @@ export const updateUser: RequestHandler = async (req, res, next) => {
         return res.json({ id });
```

Comment:
same here abot use next() and everywhere where it is possible

Reply:
added next (), thanks for explanations